### DISCUSSION TITLE
Change the installer's code?

### DISCUSSION THREAD

**Discussion by choneas** [AutoDarkMode/Windows-Auto-Night-Mode#466] [Q&A] [answered]
I want to contribute some new languages to the software installer.

**Comment by Armin2208** [comment, marked as answer]
Thanks for your support. No need to translate our installer. We use Inno Setup as installer and as I saw: there are many unofficial translation files available, also Chinese! So I will put the missing languages into the installer with the next update.

Thanks again.
